src/module_MEDIATOR.F90

```diff
@@ -4890,8 +4895,17 @@ subroutine MedPhase_atm_ocn_flux(gcomp, rc)
     type(InternalState)         :: is_local
     integer                     :: i,j,n
     character(ESMF_MAXSTR) ,pointer  :: fieldNameList(:)
+
+! LHC 2020
+#ifndef DATM
```

Comment:
The existing option for using the aofluxes is #ifdef FV3_CPLD; but beyond that, you should have a way of controlling which formulation to use when the aofluxes are actually being calculated.